src/ExternalApiExamples/appsettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "TokenProvider": {
+    "AuthorizationScope": "",
+    "ClientId": "",
+    "ClientSecret": ""
+  },
+  "StudicaExternalApiBaseAddress": "",
```

Comment:
We can probably set this to "https://gateway.kmdlogic.io/studica-review"?

Reply:
fixed